roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +36,33 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
-- name: start netmaster
-  shell: systemctl daemon-reload && systemctl start netmaster
+- name: restart netmaster
```

Comment:
we can use the `service` module here, that will simplify this task.

Also I am wondering if always restarting netmaster is desirable in a production setup. May be this task needs to restart only if a new netmaster binary was copied?